crates/ruma-events/CHANGELOG.md

```diff
@@ -32,6 +32,10 @@ Improvements:
 - `RoomMessageEventContent::set_mentions` is deprecated and replaced by
   `add_mentions` that should be called before `make_replacement`.
 
+Breaking changes:
```

Comment:
There is no need for this in the changelog, those types were not available in the last release of Ruma.

If you want to put anything in the changelog it would be the bigger changes from before, like that `CallMemberEventContent` is now an enum, but it should go under "Unreleased" not under "0.28.1". Also it would not be considered a breaking change because this is behind and unstable feature.

Reply:
It was not an enum in the latest release, and the state key was not a `String` yet: https://docs.rs/ruma-events/0.28.1/ruma_events/call/member/struct.CallMemberEventContent.html

Reply:
Oh you are talking about the initial transition to legacy + session call member events.
I see.

Reply:
Yes, what we need in the changelog are changes since the last release.

Reply:
I updated the changelog to also include the (older) change to support Legacy and Session membership contents.